### PR TITLE
Add missing space

### PR DESCRIPTION
**Pull request by APinfinite**
### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/admin/overview/about-enterprise-accounts.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ An enterprise account allows you to manage multiple organizations{% if enterpris
 
 - Member access and management (organization members, outside collaborators){% if enterpriseServerVersions contains currentVersion %}
 - Billing and usage ({% data variables.product.prodname_ghe_server %} instances, user licenses, {% data variables.large_files.product_name_short %} packs){% endif %}
-- Security{% if enterpriseServerVersions contains currentVersion %}(single sign-on, two factor authentication)
+- Security {% if enterpriseServerVersions contains currentVersion %}(single sign-on, two factor authentication)
```
Comment (Aggrord09) [inline content/admin/overview/about-enterprise-accounts.md:18]: test1

Comment (Aggrord09) [reply content/admin/overview/about-enterprise-accounts.md:18]: test

**Review by Aggrord09 — COMMENTED** [review]
test

**Comment by janiceilene** [comment]
Thanks for opening a PR @APinfinite! This is [rendering correctly](https://docs.github.com/en/enterprise-server@2.22/admin/overview/about-enterprise-accounts) with the space after the liquid statement, so I'm going to close this out 💛 

If you're looking for somewhere to contribute, there are quite a few open issues with the [help wanted](https://github.com/github/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label!

**Pull request closed without merging**